client/components/CreateProfile.js

```diff
@@ -0,0 +1,238 @@
+    })
+    .catch( (err) => console.log('Image POST error :', err));
+
+  };
+
+  _pickImage = async () => {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      allowsEditing: true,
+      aspect: [4, 3],
+      base64: true
+    });
+
+
+    // console.log(result);
+    
+    if (!result.cancelled) {
+      // this.setState({ image: result.uri });
+      this.setState({ image: result.uri, base64: result.base64 });
+      // console.log(converted);
+    }
+  };
+  
+  render() {
+    const { image } = this.state;
+    return (
+      <View style={styles.container}>
+
+        <Formik
+          initialValues={{ fName: '', lName: '', email: '', occupation: '', date: '' }}
+
+          onSubmit={ async ({fName, lName, email, occupation, date}) => {
+            
+            Keyboard.dismiss();
+            await this.postImage(this.state.base64, fName);
+            const { uploadedImageURI } = this.state;
+
+            let newProfile = createNewProfile(uploadedImageURI, fName, lName, email, occupation, date);
+
+            this.props.navigation.navigate('CreateProfileAddInfo', {
+              newProfile: newProfile
+            });
+          }
+          }>
+          {({ handleChange, handleSubmit, values }) => (
+
+
+            <ScrollView 
+              contentContainerStyle={styles.wrapper}
+              showsVerticalScrollIndicator={false}
+              >
+
+              <Text style={{ fontSize: 12, marginBottom: 30 }}>Photo</Text>
+              <View style={{ flex: 1, justifyContent: 'center' }}>
+                {image &&
+                  <Avatar
+                    onPress={this._pickImage}
+                    source={{ uri: image }}
+                    large
+                    avatarStyle={{ width: 110, height: 110, marginBottom: 20 }}
+                    rounded
+                  />}
+              </View>
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('fName')}
+                value={values.fName}
+                label="First Name"
+                placeholder="First Name"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('lName')}
+                value={values.lName}
+                label="Last Name"
+                placeholder="Last Name"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('email')}
+                value={values.email}
+                label="Email"
+                placeholder="Email"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('occupation')}
+                value={values.occupation}
+                label="Occupation"
+                placeholder="Occupation"
+              />
+
+
+              <DatePicker
```

Comment:
you could make a separate component for this and then just import it and use it from here, to help with code size.